packages/babel-parser/src/parser/statement.js

```diff
@@ -1406,6 +1406,9 @@ export default class StatementParser extends ExpressionParser {
         if (state.hadConstructor && !this.hasPlugin("typescript")) {
           this.raise(key.start, Errors.DuplicateConstructor);
         }
+        if (isConstructor && this.hasPlugin("typescript") && member.override) {
+          this.raise(key.start, Errors.OverrideOnConstructor);
```

Comment:
TS nightly does not throw on
```ts
class C extends class {} {
  override constructor() { super() };
}
```

https://www.typescriptlang.org/play?ts=4.3.0-dev.20210406#code/MYGwhgzhAEDC0FMAeAXBA7AJjUkYG8BfafAKGmgHsA3BAJzoEtMFphL0IU6BXYFSnQAUAShLQIPAA71R0QgG5ShIA

Reply:
Not right. This is a bug of TypeScript and they fixed it just now: https://github.com/microsoft/TypeScript/commit/f526a38856db6f579beeb883db967963b029f888 .